src/extensions/query-translator/clients/const.ts

```diff
@@ -0,0 +1,104 @@
+  Treemap: `MATCH path=(:Company{name:'NeoDash'})-[:HAS_DEPARTMENT*]->(:Department) WITH nodes(path) as no WITH no, last(no) as leaf WITH  [n IN no[..-1] | n.name] AS result, sum(leaf.employees) as val RETURN result, val`,
+  'Radar Chart': `MATCH (s:Skill) 
+  MATCH (:Player{name:"Messi"})-[h1:HAS_SKILL]->(s) 
+  MATCH (:Player{name:"Mbappe"})-[h2:HAS_SKILL]->(s) 
+  MATCH (:Player{name:"Benzema"})-[h3:HAS_SKILL]->(s) 
+  MATCH (:Player{name:"C Ronaldo"})-[h4:HAS_SKILL]->(s) 
+  MATCH (:Player{name:"Lewandowski"})-[h5:HAS_SKILL]->(s) 
+  RETURN s.name as Skill, h1.value as Messi, h2.value as Mbappe, h3.value as Benzema,  h4.value as Ronaldo, h5.value as Lewandowski`,
+  'Sankey Chart': 'MATCH (p:Person)-[r:RATES]->(m:Movie) RETURN p, r, m',
+  Map: 'MATCH (b:Brewery) RETURN b',
+  'Single Value': 'MATCH (n) RETURN COUNT(n)',
+  'Gauge Chart': 'MATCH (c:CPU) WHERE c.id = 1 RETURN c.load_percentage * 100',
+  'Raw JSON': 'MATCH (n) RETURN COUNT(n)',
+  'Pie Chart': 'Match (n:Person)-[e]->(m:Movie) RETURN m.title as Title, COUNT(p) as People LIMIT 10',
+};
+
+export const TASK_DEFINITION = `Task: Generate Cypher queries to query a Neo4j graph database based on the provided schema definition. These queries will be used inside NeoDash reports.
+Documentation for NeoDash is here : https://neo4j.com/labs/neodash/2.2/
+Instructions:
+Use only the provided relationship types and properties.
+Do not use any other relationship types or properties that are not provided.
+The Cypher RETURN clause must contained certain variables, based on the report type asked for.
+Report types :
+Table - Multiple variables representing property values of nodes and relationships.
+Graph - Multiple variables representing nodes objects and relationships objects inside the graph.
+Bar Chart - Two variables named category(a String value) and value(numeric value).
+Line Chart - Two numeric variables named x and y.
+Sunburst - Two variables named Path(list of strings) and value(a numerical value).
+Circle Packing - Two variables named Path(a list of strings) and value(a numerical value).
+Choropleth - Two variables named code(a String value) and value(a numerical value).
+Area Map - Two variables named code(a String value) and value(a numerical value).
+Treemap - Two variables named Path(a list of strings) and value(a numerical value).
+Radar Chart - Multiple variables representing property values of nodes and relationships.
+Sankey Chart - Three variables, two being a node object (and not a property value) and one representing a relationship object (and not a property value).
+Map - multiple variables representing nodes objects(should contain spatial propeties) and relationship objects.
+Single Value - A single value of a single variable.
+Gauge Chart - A single value of a single variable.
+Raw JSON - The Cypher query must return a JSON object that will be displayed as raw JSON data.
+Pie Chart - Two variables named category and value.`;
+
+export const MAX_NUM_VALIDATION = 3;
```

Comment:
we should also document this somewhere in the app itself.
perhaps in the config screen for the extension.
So the user knows we make between 1 and 3 calls per translation request.